source/operators/pdb-management/examples/demo/scripts/demo-auth-service.sh

```diff
@@ -0,0 +1,37 @@
+
+  #!/bin/bash
+  echo "🚀 Starting Auth Service Demo..."
+  echo "----------------------------------------"
+
+  # Clean up first
+  echo "1⃣ Cleaning up any existing deployment..."
+  kubectl delete deployment auth-service -n pdb-demo 2>/dev/null
+  sleep 2
+
+  # Start log monitoring in background
+  echo "2⃣ Starting log monitor..."
+  kubectl logs -n canvas deployment/pdb-management-controller-manager -f --tail=0 | \
```

Comment:
Using the helm chart oda-canvas the operator name is `canvas-pdb-management-operator` instead of `pdb-management-controller-manager`.

Also in other places the deployment name `pdb-management-controller-manager`is used and should be replaced with `canvas-pdb-management-operator`